tululbot/commands.py

```diff
@@ -65,6 +65,20 @@ def hotline(message):
         return bot.forward_message(message.chat.id, message.chat.id, HOTLINE_MESSAGE_ID)
 
 
+@bot.message_handler(is_reply_to_bot='Siapa yang ultah?')
+@bot.message_handler(commands=['hbd'])
+def hbd(message):
+    app.logger.debug('Detected as hbd command')
+    term = _extract_birthday_boy_or_girl_name(message)
+    if not term:
+        return bot.reply_to(message, 'Siapa yang ultah?',
+                            force_reply=True)
+    else:
+        greetings = "hoi " + term \
+            + " met ultah ya moga sehat dan sukses selalu :tada: :confetti_ball:"
```

Comment:
`:tada:` dkk itu client yang convert (in your case web telegram)

pake UTF-8

https://github.com/tulul/lycantulul_bot/commit/0aeaff016631fa821c894a03ab2bb8cb2ec922aa#diff-7c846ee65937ae29836d842a7df056d8R6
http://apps.timwhitlock.info/emoji/tables/unicode